code/modules/paperwork/papershredder.dm

```diff
@@ -41,16 +41,23 @@
 				to_chat(user, span("warning", "\The [src] must be anchored to the ground to operate!"))
 				return
 			if(paperamount == max_paper)
-				to_chat(user, "<span class='warning'>\The [src] is full; please empty it before you continue.</span>")
+				to_chat(user, span("warning", "\The [src] is full; please empty it before you continue."))
 				return
 			if (paper_result > 0)
```

Comment:
```suggestion
			if (paper_result > 0)
```

This check is unnecessary, I guess. Since `paper_result` can't be less than zero and it doesn't matter if it is zero, we can just have the following line by itself?